pkgs/epnix/support/adsDriver/default.nix

```
@@ -0,0 +1,36 @@
+{
+  mkEpicsPackage,
+  fetchFromGitHub,
+  epnix,
+  boost,
+  epnixLib,
+  lib,
+}:
+mkEpicsPackage rec {
+  pname = "adsDriver";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Cosylab";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-Ruzi+H8MmIgv23pzFXZlvkk3HtbDzQ9LTTVzmeGWrSI==";
+  };
+
+  buildInputs = [ boost ];
```

So EPICS cross-compilation is weird, and will compile each project twice: once for the build architecture, and once for the host architecture.

This implies that usually, build dependencies are needed twice, unfortunately…

```suggestion
  nativeBuildInputs = [ boost ];
  buildInputs = [ boost ];
```

---

ah okay, yeah, hadn't tested with cross-compiling yet